doc/user/bgp.rst

```diff
@@ -2573,6 +2573,20 @@ are reached using *core* MPLS labels which are distributed using LDP or BGP
 labeled unicast.  *bgpd* also supports inter-VRF route leaking.
 
 
+L3VPN over GRE interfraces
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is possible to configure and assign an incoming route-map to L3VPN peer,
+that permits BGP updates to be installed over a GRE interface instead of an
+MPLS based backbone. Command under route-map is the following one:
```

Comment:
Why say anything about MPLS here?

Reply:
MPLS is part of the algorithm: if the path of the next hop has no MPLS label configured, then it is not possible to install that entry.
Do you mean I should generalize with segment-routing case (including srv6?) ? I hesitate to enter this, because I did not test this.

Reply:
Again, I think the implications, from the user perspective not developer perspective, should be spelled out.

Reply:
MPLS => I generalised to SRv6

Reply:
> MPLS => I generalised to SRv6
> user perspective => I briefly recommended some config elements on remote endpoint